tests/scripts/thread-cert/border_router/test_multi_border_routers.py

```diff
@@ -170,7 +170,7 @@ def test(self):
 
         br1.disable_br()
 
-        self.simulator.go(15)
+        self.simulator.go(315)
```

Comment:
I think the prefix related tests on netdata can be done sooner (after a wait period of say 15 sec).
The SLAAC address related tests can be done after than (after an additional time of say 300 sec).